packages/elm/index.js

```diff
@@ -14,9 +14,10 @@ module.exports = elm
  * @param {string}                  [options.cwd]           Custom location for your elm files.
  * @return {Function}
  */
-function elm (options, isProduction) {
-  options = options || {}
-  isProduction = isProduction || process.env.NODE_ENV === 'production'
+function elm (options = {}, isProduction) {
```

Comment:
I didn't write the block and I agree, a more elegant solution than a isProduction flag would be nice. (The bad boy that I am) I fixed a small issue I recognized on-the-fly while doing the node 6+ thing, even though it's not related.

But I think we can live with it for now. Not sure how many people currently use the block. I would say let's improve it once we know it is really used?